karate-core/src/main/java/com/intuit/karate/core/ScenarioEngine.java

```diff
@@ -491,6 +491,9 @@ private void multiPartInternal(String name, Object value) {
         if (name != null) {
             map.put("name", name);
         }
+        if(value instanceof Integer || value instanceof Float || value instanceof Double) {
```

Comment:
can you see if `value instanceof Number` would be sufficient ?

Reply:
Yes, that was enough. Thank you, I modified the code in the same PR.